targets/TARGET_Atmel/TARGET_SAM_CortexM0P/utils/cmsis/TARGET_SAML21/source/system_saml21.c

```diff
@@ -50,7 +50,7 @@
  * Initial system clock frequency. The System RC Oscillator (RCSYS) provides
  *  the source for the main clock at chip startup.
  */
-#define __SYSTEM_CLOCK    (4000000)
+#define __SYSTEM_CLOCK    (4000000 * 12)
```

Comment:
The comment indicates that this is the "System RC Oscillator" which is typically internal to the chip. Why is the being multiplied by 12? If a clock needs to be updated shouldn't it be for an external or PLL clock?

Reply:
Good question, the previous version ran on 4 MHz. In this PR we're upping the clock frequency to 48 MHz (normal speed of the device). However, if this define is not in sync with the actual clock speed all timers are off in Mbed OS. But I cannot update this value after boot (I've tried but it doesn't do anything), so I think it should be the value after boot here, not the one during boot. Have to update the comment though.